poetry/core/packages/package.py

```diff
@@ -299,6 +299,7 @@ def add_dependency(
                     rev=constraint.get("rev", None),
                     category=category,
                     optional=optional,
+                    develop=constraint.get("develop", True),
```

Comment:
Should this be `True` by default here? Not sure what the general expectation is.

Reply:
I wanted to set it to `False` by default but since the current behavior is similar to `True`, I don't want to break things if people rely on it, even though it's not documented anywhere and, as such, they should not rely on it.